pci/src/vfio.rs

```diff
@@ -876,27 +876,51 @@ impl VfioCommon {
         });
     }
 
+    /// Returns true, if the device claims that is has a PCI capability list.
+    pub(crate) fn has_capabilities(&self) -> bool {
```

Comment:
I'm not sure why all functions are public. I followed the convention and made it public as well.

Reply:
Good callout. I *think* the reason why many of the functions are made public to the crate originally because they were used by another module `vfio_user` which now is a separate crate. So I think it would be good to revisit these pub(crate) functions and make them internal to the vfio module if not being used elsewhere. Would you please add another commit to address this issue. Thank you.

Reply:
I've added a commit to reduce visibility of functions in `VfioCommon` that are not used externally.